ht-11/index.js

```diff
@@ -0,0 +1,68 @@
+function debounce (func, ms) {
+  let timer;
+  return function (...args) {
+    clearTimeout(timer);
+    timer = setTimeout (() => {
+      func.apply(this, args)
+    }, ms);
+  }
+};
+
+function search() {
+  const body = document.querySelector('.img-wrapper');
+  const searchInput = document.getElementById("search-input");
+  let pageN = 1;
+  let searchValue;
+
+  function removeMarkup () {
+    Array.from(document.querySelectorAll(`[data-search='res']`)).forEach(element => {
+      element.remove();
+    });
+  };
+
+  searchInput.addEventListener('input', debounce (() => {
+    searchValue = searchInput.value.replace( / /g, "+" );
+    if(searchValue) {
+      removeMarkup();
+      fetchFunc();
+    }
+  }, 1500));
+
+  function fetchFunc() {
+    const res = fetch(`https://pixabay.com/api/?key=34969259-1340cd29ea32dd0019bae9b64&q=${searchValue}&image_type=photo&page=${pageN}&per_page=4`)
```

Comment:
const baseUrl = 'https://pixabay.com/api/'
const apiKey = '34969259-1340cd29ea32dd0019bae9b64' (also it is not a good practice to keep api key in the repo)